README.md

```diff
@@ -2,43 +2,21 @@
 [![Build Status](https://travis-ci.org/haskell/ghcup.svg?branch=master)](https://travis-ci.org/haskell/ghcup)
 [![license](https://img.shields.io/github/license/haskell/ghcup.svg)](COPYING)
 
-# GHC up
+Although [`cabal-install`](https://hackage.haskell.org/package/cabal-install) manages Haskell library dependencies and can select a specific version of `ghc` (see [`with-compiler`](https://cabal.readthedocs.io/en/latest/nix-local-build.html#cfg-flag---with-compiler)), it will fail if the required version of `ghc` is not available. Unfortunately, many operating systems do not offer a way to install specific versions of `ghc`.
 
-Installs a specified GHC version into `~/.ghcup/ghc/<ver>`,
-and places `ghc-<ver>` etc. symlinks in `~/.ghcup/bin/`.
-Additionally allows to manage currently selected ghc
-via unversioned symlinks.
+`ghcup` makes it easy to install specific versions of `ghc` on GNU/Linux, and can also bootstrap a fresh Haskell developer environment from scratch.
 
-This uses precompiled GHC binaries that have been
-compiled on fedora/debian by
-[upstream GHC](https://www.haskell.org/ghc/download_ghc_8_6_1.html#binaries).
+Inspired by [rustup](https://github.com/rust-lang-nursery/rustup.rs), [pyenv](https://github.com/pyenv/pyenv) and [jenv](http://www.jenv.be).
```

Comment:
I don't know pyenv and jens. It was really only inspired by rustup ;)

Reply:
it inspired some of my suggestions, and my `ghc-env` previously :smile: 

I think it's also important to point out that this is similar in nature to `pyenv` and `jenv` as anybody who knows those tools will immediately understand the point.

Reply:
I was only inspired by rustup. I don't know how specific we want to be here. Especially the `--help` design I mostly stole from rustup, so there is some direct correlation. But there is no intended correlation to pyenv or jenv.